app/controllers/groups/create.js

```diff
@@ -0,0 +1,48 @@
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+
+export default class extends Controller {
+
+  @service errorHandler;
+
+  groupName ='';
```

Comment:
Use `group.name`

Reply:
but i am creating it in controller, so on first time there is no model.group.name?